NaviGator/perception/navigator_vision/nodes/vrx_classifier.py

```diff
@@ -80,8 +84,13 @@ def set_enable_srv(self, req):
         return {'success': True}
 
     def image_cb(self, msg):
+        self.last_image = msg
         return
 
+    def taskinfoSubscriber(self, msg):
+        if not self.is_perception_task and msg.name == "perception":
```

Comment:
This doesn't allow the state to ever go `false`. Consider the following:

```suggestion
   self.is_perception_task = msg.name == "perception"
```

Reply:
Thanks!